tests/rkt_net_test.go

```diff
@@ -608,6 +608,10 @@ func testNetCustomNatConnectivity(t *testing.T, nt networkTemplateT) {
 
 	// Child connects to host
 	hostname, err := os.Hostname()
+	if err != nil {
+		t.Fatalf("os.Hostname() got error %q", err)
```

Comment:
Yeah I don't really understand your argument about it being harder to fix or debug. If `os.Hostname()` fails something is fundamentally broken on the host or in the Go runtime. It's not a test failure for us to fix.